website/docs/r/linux_web_app.html.markdown

```diff
@@ -0,0 +1,555 @@
+
+* `auto_swap_slot_name` - (Optional) The Linux Web App Slot Name to automatically swap to when deployment to that slot is successfully completed.
+
+* `cors` - (Optional) A `cors` block as defined above.
+
+* `default_documents` - (Optional) Specifies a list of Default Documents for the Linux Web App.
+
+* `ftps_state` - (Optional) The State of FTP / FTPS service. Possible values include: `AllAllowed`, `FtpsOnly`, `Disabled`.
+
+* `health_check_path` - (Optional) The path to the Health Check.
+
+* `http2_enabled` - (Optional) Should the HTTP2 be enabled?
+
+* `ip_restriction` - (Optional) One or more `ip_restriction` blocks as defined above.
+
+* `load_balancing_mode` - (Optional) The Site load balancing. Possible values include: `WeightedRoundRobin`, `LeastRequests`, `LeastResponseTime`, `WeightedTotalTraffic`, `RequestHash`, `PerSiteRoundRobin`. Defaults to `LeastRequests` if omitted.
+
+* `local_mysql` - (Optional) Use Local MySQL. Defaults to `false`.
+
+* `managed_pipeline_mode` - (Optional) Managed pipeline mode. Possible values include: `Integrated`, `Classic`.
+
+* `minimum_tls_version` - (Optional) The configures the minimum version of TLS required for SSL requests. Possible values include: `1.0`, `1.1`, and  `1.2`.
+
+* `number_of_workers` - (Optional) The number of Workers for this Linux App Service.
+
+* `remote_debugging` - (Optional) Should Remote Debugging be enabled. Defaults to `false`.
+
+* `remote_debugging_version` - (Optional) The Remote Debugging Version. Possible values include `VS2017` and `VS2019`
+
+* `scm_ip_restriction` - (Optional) One or more `scm_ip_restriction` blocks as defined above.
+
+* `scm_minimum_tls_version` - (Optional) The configures the minimum version of TLS required for SSL requests to the SCM site Possible values include: `1.0`, `1.1`, and  `1.2`.
+
+* `scm_use_main_ip_restriction` - (Optional) Should the Linux Web App `ip_restriction` configuration be used for the SCM also.
+
+* `use_32_bit_worker` - (Optional) Should the Linux Web App use a 32-bit worker.
+
+* `websockets` - (Optional) Should Web Sockets be enabled. Defaults to `false`.
+
+---
+
+A `slow_request` block supports the following:
+
+* `count` - (Required) The number of Slow Requests in the time `interval` to trigger this rule.
+
+* `interval` - (Required) The time interval in the form `hh:mm:ss`.
+
+* `time_taken` - (Required) The threshold of time passed to qualify as a Slow Request in `hh:mm:ss`.
+
+* `path` - (Optional) The path for which this slow request rule applies.
+
+---
+
+A `status_code` block supports the following:
+
+* `count` - (Required) The number of occurrences of the defined `status_code` in the specified `interval` on which to trigger this rule.
+
+* `interval` - (Required) The time interval in the form `hh:mm:ss`.
+
+* `status_code_range` - (Required) The status code for this rule, accepts single status codes and status code ranges. e.g. `500` or `400-499`. Possible values are integers between `101` and `599`
```

Comment:
this takes both ranges and single values?
```suggestion
* `status_codes` - (Required) The status code for this rule, accepts single status codes and status code ranges. e.g. `500` or `400-499`. Possible values are integers between `101` and `599`
```

Reply:
is this also a list?

Reply:
This is explicitly a range string, rather than a list. It cannot take individual codes, only a `x` or `x-y` format. (values such as `x, y, z` are not accepted). I felt range better described this behaviour than simply codes?